tests/DataObjectTest.py

```diff
@@ -110,3 +114,26 @@ def __init__(self, a=None, **kw):
     def test_rdfs_comment_property(self):
         a = DataObject(rdfs_comment='Hello')
         self.assertIn('Hello', a.rdfs_comment())
+
+    def test_context_getter(self):
+        a = DataObject()
+        self.assertIsNone(a.context)
+
+    def test_context_setter(self):
+        a = DataObject()
+        a.context = 42
+        self.assertEquals(a.context, 42)
+
+    def test_dataobject_property_that_generate_partial_property(self):
+        for property_classmethod in dataobject_properties_methods():
+            partial_property = DataObject().__getattribute__(property_classmethod)(dict())
```

Comment:
You could test this same test by providing no arguments to the property classmethod.

Reply:
I think you're looking for `getattr(DataObject, property_classmethod)` here.